CONTRIBUTING.md

```diff
@@ -18,6 +18,12 @@ get invited to meetings.
 See the [#otel-profiles](https://cloud-native.slack.com/archives/C03J794L0BV)
 slack channel for discussions and questions.
 
+## Pre-requisites
+
+- Linux (The profiler currently supports only Linux systems)
+- Windows users can also use WSL
```

Comment:
WSL2 (I expect that WSL1 will not work at all) is not a supported configuration, so we probably shouldn't mention it here as there are known issues e.g. see #24

```suggestion
```